src/BridgeExtensionProxy.sol

```diff
@@ -4,7 +4,7 @@ pragma solidity ^0.8.20;
 import "@openzeppelin/contracts/proxy/transparent/TransparentUpgradeableProxy.sol";
 
 contract BridgeExtensionProxy is TransparentUpgradeableProxy {
```

Comment:
why don't just deploy the `TransparentUpgradeableProxy` ?¿

Reply:
Used like this it's just an abstraction that doesn't cost anything. And I think a clear name is helpful.
Also it was like this, see no reason to change it.